core/config/config.go

```
@@ -361,12 +362,14 @@ func migrateAccessTokens(ctx context.Context, db pg.DB, sdb *sinkdb.DB) error {
 		switch token.Type {
 		case "client":
 			grant.Policy = "client-readwrite"
+			clientGrants = append(clientGrants, &grant)
 		case "network":
 			grant.Policy = "crosscore"
+			networkGrants = append(networkGrants, &grant)
 		}
-		stores = append(stores, store.Save(ctx, &grant))
 	}
-	err = sdb.Exec(ctx, stores...)
+	err = sdb.Exec(ctx, store.SaveAll(ctx, clientGrants, "client-readwrite"))
```

It looks the error is being ignored here, but the fix is easy:

```go
err = sdb.Exec(ctx,
	store.SaveAll(ctx, clientGrants, "client-readwrite"),
	store.SaveAll(ctx, networkGrants, "crosscore"),
)
```